native/cocos/core/animation/SkeletalAnimationUtils.h

```diff
@@ -25,18 +25,21 @@
 
 #pragma once
 
-#include "math/Mat4.h"
+#include <utility>
 
+#include "math/Mat4.h"
+#include "cocos/base/RefCounted.h"
+#include "cocos/base/Ptr.h"
 namespace cc {
 
 class Node;
 
-struct IJointTransform {
-    Node *node{nullptr};
+struct IJointTransform : public RefCounted {
+    IntrusivePtr<Node> node;
```

Comment:
Does IJointTransform need to own node ? Will node be released?

Reply:
No node for currently. Do I need to remove it?

Reply:
IJointTransform is put into a global `vector`, it's very dangerous to hold the Node in a global container.
It's better to keep weak reference for `node`.

Reply:
got it